validation/src/blockchain/eth_rpc/types.rs

```diff
@@ -101,36 +93,25 @@ impl FromStr for EthProofType {
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
-/// A proof of time on the blockchain
-pub struct TimeProof {
+/// A proof of time derived from state on the blockchain
+pub struct TimeProofResults {
```

Comment:
We have lots of names for what seem like the same thing, can we consolidate on a consistent naming?

For example we have AnchorProof, TimeProofResults and chain_inclusion_proof. What is the difference between anchor, time, and chain inclusion? Do we really have three different types of proofs? Why is the function named `chain_inclusion_proof` when it returns a TimeProofResults?

Should the names instead be, AnchorProof, ChainInclusionProof and get_chain_inclusion_proof? Where anchor proof is a chain agnostic chain inclusion proof?  I'd be fine with other names too, its just not clear how all these thing relate given their current names and the differences imply there are more entities than there really are.

Reply:
yep agreed.  Took your suggested names, I think those are clearer.